Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -479,20 +479,14 @@
     state: pda-library
 
 - type: entity
-  parent: BaseSecurityPDA
+  parent: BaseJusticePDA # DeltaV - Added secwatch to justice. Was BaseSecurityPDA.
   id: LawyerPDA
   name: lawyer PDA
   description: For lawyers to poach dubious clients.
   components:
   - type: Pda
     id: LawyerIDCard
     state: pda-lawyer
-    penSlot:
```

Comment:
should have a comment saying penSlot was removed as its in parent

Reply:
I forgot, will do.